### DISCUSSION TITLE
Spatie Translatable Improvements suggestion.

### DISCUSSION THREAD

**Discussion by AlexisSerneels** [filamentphp/filament#1117] [Ideas]
Hi,

Thank you for this awesome package. I would like to code and PR some improvements about the spatie/translatable integration and wanted to check with you if those improvements were welcomed in your code base or if I should simply do them on my own repo.

- Improve `ListRecords` to add a language selector and be able to see translated content.
- Improve `EditRecord` to be able to configure whether to fallback to default language, which is the current behavior. If the fallback locale is displayed, improve the input so it indicates that the value is imported from fallback. Current implementation causes confusion imo. On top of that, the fallback value should not be saved on update (as it is now)
- Improve `EditRecord` so the values are not saved when simply switching locales (To compare version by example)

Let me know

**Comment by danharrin** [comment]
Hey, thanks for your suggestions

- I'd definitely like a language selector for the List page, thank you.
- Yeah, we should just add an extra method to get which language to fallback to, which you can override.
- The third improvement - I think it could cause issues as we then need to keep track of multiple translated versions of the record in one Livewire component and sync untranslated changes between them. Unless you can find a really clean way of doing this, I'm gonna say no for now.

Looking forward to the PRs!

**Comment by danharrin** [comment]
#2005 adds list page translations :)
